bundles/org.openhab.binding.ferroamp/src/main/java/org/openhab/binding/ferroamp/internal/EhubParameters.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ferroamp.internal;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link EhubParameters} is responsible for all parameters regarded to EHUB
+ *
+ * @author Örjan Backsell - Initial contribution
+ *
+ */
+
+@NonNullByDefault
+public class EhubParameters {
+    public String jsonPostEhub;
+
+    public EhubParameters(String jsonPostEhub) {
+        this.jsonPostEhub = jsonPostEhub;
+    }
+
+    public static List<String> getChannelParametersEhub() {
+        final List<String> channelParametersEhub = Arrays.asList("grid-frequency", "ace-current-l1", "ace-current-l2",
```

Comment:
20241107
Deleted the class. Has been used earlier, but I have missed to delete it.

Reply:
Did you forget to push these changes as it is still there.

Reply:
20241115
I had a look at channel-groups, but I think it look like it's the very same lines just split into several xml files instead of one thing-types.xml

Reply:
Gone